website/docs/docs/trusted-adapters.md

```diff
@@ -39,3 +31,12 @@ The following are **Trusted adapters** ✓ you can connect to in dbt Core:
 import AdaptersTrusted from '/snippets/_adapters-trusted.md';
 
 <AdaptersTrusted />
+
+<MSCallout />
+
+Here are the verified data platforms that connect to dbt and its latest version.
+
+import AdaptersVerified from '/snippets/_adapters-verified.md';
```

Comment:
This snippet still uses the verified language in the Azure block. Should we change that here?
<img width="546" alt="Screenshot 2024-02-12 at 11 48 03 AM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/fcc22dab-e818-44e7-b9bb-5293537b4a62">

Reply:
How does one remove this?

Reply:
thanks for flagging! it's now removed